### PR TITLE
Zhang peng ppp

### PR DESCRIPTION
**Pull request by icytornado**

### CODE REVIEW

File: docs/team/icytornado.md [hunk 1]
```diff
@@ -5,16 +5,55 @@ title: Zhang Peng's Project Portfolio Page
...
+        - `edit-appt` command allow the user to edit an existing appointment by changing the fields for the appointment.
+        - `edit-appt` + `index` + `field` to edit => edit the specified field of the appointment specified by the appointment index.
+        - the `field` to edit is preceded by its related prefix, i.e. `pt/` for patient, `dr/` for doctor, `at/` for appointment start date and time, `to/` for appointment end date and time, `dur/` for appointment duration, `/t` for appointment tags.
+    
+  * Implementation of the `Doctor` feature
+    - *Description*:
+        - Created the `Doctor` class and implemented its related command features including
+          `delete-doctor`, `edit-doctor`, `add-doctor`, `clear-doctor`, `list-doctor`.
+            
+  * Added some relevant test classes for `Doctor`.
```
Comment (onnwards) [inline docs/team/icytornado.md]: i think it is only fair to mention here that there were a lot of issues with the implementation, and other members of the team had to step in to resolve those issues.

File: docs/team/icytornado.md [hunk 2]
```diff
@@ -5,16 +5,55 @@ title: Zhang Peng's Project Portfolio Page
...
+    - Updated the UML diagrams and description for the Model component under the Design section:
+      * [Model Class Diagram](https://github.com/AY2021S2-CS2103-W17-2/tp/blob/master/docs/diagrams/ModelClassDiagram.puml)
+      * [Better Person Model Class Diagram](https://github.com/AY2021S2-CS2103-W17-2/tp/blob/master/docs/diagrams/BetterPersonModelClassDiagram.puml)
+      * [Better Appointment Model Class Diagram](https://github.com/AY2021S2-CS2103-W17-2/tp/blob/master/docs/diagrams/BetterAppointmentModelClassDiagram.puml)
+        - Pull request [#163](https://github.com/AY2021S2-CS2103-W17-2/tp/pull/163)
+    - Added user stories for some command features.
+       - Pull request [#154](https://github.com/AY2021S2-CS2103-W17-2/tp/pull/154)
+    - Added Use Case documentation on `edit-patient`, `find-patient`, `delete-doctor`, `edit-doctor`, `find-doctor`, `edit-appt` features.
+       - Pull request [#162](https://github.com/AY2021S2-CS2103-W17-2/tp/pull/162)
+    - Added Instructions for Manual Testing for `edit-patient`, `find-patient`, `delete-doctor`, `edit-doctor`, `find-doctor`, `edit-appt` features.
```
Comment (onnwards) [inline docs/team/icytornado.md]: `delete-doctor` was not added by you.

File: docs/team/icytornado.md [hunk 3]
```diff
@@ -5,16 +5,55 @@ title: Zhang Peng's Project Portfolio Page
...
+       - Pull request [#169](https://github.com/AY2021S2-CS2103-W17-2/tp/pull/169)
+
+
+* **Project management**:
+   * Updated a few PRs with the relevant tags and milestones for the project
```
Comment (onnwards) [inline docs/team/icytornado.md]: these PRs that you updated were opened by you and were supposed to be tagged by you anyway, and not part of project management

File: docs/team/icytornado.md [hunk 4]
```diff
@@ -5,16 +5,55 @@ title: Zhang Peng's Project Portfolio Page
...
+   * Closed and merged a few PRs.
```
Comment (onnwards) [inline docs/team/icytornado.md]: the PRs that you merged were yours to begin with. the only exception to that is a PR made by jacob, which was to help you fix bugs in the doctor implementation. i do not believe that this counts as project management.

**Review by onnwards — CHANGES_REQUESTED** [review]

File: docs/team/icytornado.md (hunk 4 above)
Comment (icytornado) [reply docs/team/icytornado.md]: will update accordingly

File: docs/team/icytornado.md (hunk 3 above)
Comment (icytornado) [reply docs/team/icytornado.md]: will update accordingly

File: docs/team/icytornado.md (hunk 1 above)
Comment (icytornado) [reply docs/team/icytornado.md]: i think i just say what i added in.

**Review by onnwards — APPROVED** [review]

**Pull request merged**